### DISCUSSION TITLE
Does Immich Work with Redis 7?

### DISCUSSION THREAD

**Discussion by vamega** [immich-app/immich#9524] [Q&A] [answered]
I see there are some automated PR's to nump the tag of to Redis 7.2 that were closed.
Is that because immich is known to be incompatible with Redis 7.2?

**Comment by bo0tzz** [comment, marked as answer]
We haven't tested compatibility, and I can't imagine there would be any issues. We closed the PRs because we're planning to get rid of redis eventually so there's no real point in updating it.
